samples/hssi/hssi_10g_cmd.h

```diff
@@ -252,6 +267,96 @@ class hssi_10g_cmd : public hssi_cmd
     return test_afu::success;
   }
 
+  void hssi_perf(hssi_afu *hafu)
+  {
+    fpga_properties pprops = NULL;
+    fpga_result res = FPGA_OK;
+    uint32_t num_matches = 1;
+    uint64_t bitstream_id;
+    double clk_freq = 0;    
+    uint32_t major = 0;
+    fpga_token token;  
+
+    std::cout << "HSSI performance: " << std::endl;
+
+    res = fpgaGetProperties(NULL, &pprops);
+    if (res != FPGA_OK) {
+    	std::cerr << "Failed to get properties" << std::endl;
+    	return;
+    }
+
+    res = fpgaPropertiesSetObjectType(pprops, FPGA_DEVICE);
+    if (res != FPGA_OK) {
+    	std::cerr << "Failed to set type to FPGA_DEVICE" << std::endl;
+    	return;
+    }
+
+    fpgaEnumerate(&pprops, 1, &token, 1, &num_matches);
```

Comment:
Couple of issues here:

1. You're assuming that there is only 1 accelerator device in the system.  This is not a safe assumption.
2. You're ignoring the return value of fpgaEnumerate.  In general, that is not a safe thing to do.